crds/doc-ru-virtualdisksnapshots.yaml

```
@@ -15,7 +15,7 @@ spec:
                     Имя виртуального диска, для которого будет создан снимок.
                 volumeSnapshotClassName:
                   description: |
-                    Имя класса снимков томов, который будет использоваться при создании снимка виртуального диска.
+                    DEPRECATED: Данное поле устарело и будет удалено в будущем релизе
```

```suggestion
                    DEPRECATED: Данное поле устарело и будет удалено в будущем релизе
```
Please add `x-doc-deprecated: true` in the main spec.